src/styles/blocks/card.scss

```diff
@@ -72,13 +72,13 @@
     font-weight: $font-weight-bolder;
     justify-content: center;
     align-items: center;
-    color: #fff;
+    color: $hover-color;
     text-transform: uppercase;
     border: 1px solid #ccc transparent;
```

Comment:
Add color to variables

Reply:
You can't use 4th parameter here.
https://developer.mozilla.org/en-US/docs/Web/CSS/border

Reply:
Fix it

Reply:
l add color to variables:
` color: $hover-color;`